calico/felix/plugins/fiptgenerator.py

```diff
@@ -1006,14 +1038,21 @@ def _rule_to_iptables_fragments_inner(self, chain_name, rule, ip_version,
         rule_spec_str = " ".join(str(x) for x in rule_spec if x != "")
 
         if ipt_target == "DROP":
-            rules = self.drop_rules(ip_version, chain_name, rule_spec_str, None)
+            rules = self.logged_drop_rules(ip_version, chain_name,
+                                           rule_spec_str,
+                                           log_pfx=rule.get("log_prefix"))
         else:
             rules = [" ".join(["--append", chain_name, rule_spec_str,
                                "--jump", ipt_target])]
-            if extra_rule:
-                rules.append(extra_rule)
+            if extra_rules:
+                rules.extend(extra_rules)
         return rules
 
+    def _log_target(self, rule):
+        log_pfx = rule.get("log_prefix", "calico")
```

Comment:
I think we need a slightly more informative default here.

Perhaps 'calico-iptables-packet'?